source/administration/config-settings.rst

```diff
@@ -840,6 +840,15 @@ Nickname Attribute
 | This feature's ``config.json`` setting is ``"NicknameAttribute": ""`` with string input.                                                                             |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Position Attribute
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+(Optional) The attribute in the AD/LDAP server that will be used to populate the position field in Mattermost (typically used to describe a person's job title or role at the company). When set, users will not be able to edit their position, since it is synchronized with the LDAP server. When left blank, users can set their own Position in Account Settings.
```

Comment:
"When blank, users can set their own position in the **Account Settings** dialog."